### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
- Update version to `v0.2.0`
- Update description and name to be align wording with other plugins
- Add icon
- Add metadata
Screenshot from 2020-05-27 19-34-48

- Fix urgent bugs
- Change plugin id to `com.matttermost.google-calendar`

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-google-calendar/issues/9

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {
     "id": "calendar",
```
Comment (hanzei) [inline plugin.json:2]: `calender` might be a controversial plugin `id`. If there is any time to change it to something else, it's now e.g. `com.mattermost.google-calendar`.

Comment (levb) [reply plugin.json:2]: Add it to this PR?

Comment (hanzei) [reply plugin.json]: Done

**Comment by hanzei** [comment]
@levb @hahmadia I did update the plugin id and also the readme. Then I tested if the instructions are working and fixed some urgent issue on that way. Please take another look.

**Comment by levb** [comment]
Thanks for the fixes, too!

**Comment by hanzei** [comment]
@aaronrothschild I did try out the plugin locally and it works quite well. Listing and creating events works like a charm. If you consider this plugin important enough to invest more time, I'm help further enhance it.

**Comment by aaronrothschild** [comment]
> If you consider this plugin important enough to invest more time, I'm help further enhance it.

I was just talking with a customer today who uses Google services.  I think we'll get a good reception to this plugin.  The big question is if we use the MS Calendar backend to drive this integration as well. cc @levb

**Comment by levb** [comment]
That's a bigger question. This plugin is NOT HA ready at the moment; we should IMO release a beta version regardless, and iterate.

**Pull request merged**